pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -529,6 +539,21 @@ func isSCCAvailable(c kubernetes.Interface) (bool, error) {
 	return isResourceAvailable(c, "securitycontextconstraints", "security.openshift.io", "v1")
 }
 
+func isRunningKubernetesNMStateOperator(c k8sclient.Client) (bool, error) {
+	deployments := &appsv1.DeploymentList{}
+	err := c.List(context.TODO(), deployments, k8sclient.MatchingLabels{"app": "kubernetes-nmstate-operator"})
```

Comment:
Don't we have to specify the namespace ?

Reply:
We wouldn't necessarily know what the namespace is.